apps/dashboard/src/components/activity/constants.ts

```diff
@@ -0,0 +1,92 @@
+import { JobStatusEnum, StepTypeEnum } from '@novu/shared';
+import { RiForbidFill } from 'react-icons/ri';
+import { RiErrorWarningLine } from 'react-icons/ri';
+import { BadgeVariant } from '../primitives/badge';
+import { RiCheckboxCircleLine, RiLoader3Line, RiLoader4Fill } from 'react-icons/ri';
+import { IconType } from 'react-icons/lib';
+
+export const STEP_TYPE_LABELS: Record<StepTypeEnum, string> = {
+  [StepTypeEnum.EMAIL]: 'Email',
+  [StepTypeEnum.SMS]: 'SMS',
+  [StepTypeEnum.IN_APP]: 'In-App',
+  [StepTypeEnum.CHAT]: 'Chat',
+  [StepTypeEnum.PUSH]: 'Push',
+  [StepTypeEnum.DIGEST]: 'Digest',
+  [StepTypeEnum.DELAY]: 'Delay',
+  [StepTypeEnum.TRIGGER]: 'Trigger',
+  [StepTypeEnum.CUSTOM]: 'Custom',
+};
+
+export const STATUS_STYLES = {
+  completed: 'border-[1px] border-[#b4e6c5] bg-[#e8f9ef] text-[#b4e6c5]',
+  failed: 'border-[1px] border-[#fca5a5] bg-[#fde8e8] text-[#fca5a5]',
+  delayed: 'border-[1px] border-[#f7d794] bg-[#fef9e7] text-[#f7d794]',
+  default: 'border-[1px] border-[#e0e0e0] bg-[#f0f0f0] text-[#e0e0e0]',
```

Comment:
can we please use the color names and not hardcoded hashes?

Reply:
Yes! Just learned it can be achieved with /40 thank you! :)